index.js

```diff
@@ -250,6 +250,48 @@ SteamCommunity.prototype.oAuthLogin = function(steamguard, token, callback) {
 	}, "steamcommunity");
 };
 
+/**
+ * Sign in through Steam on a third party website via OpenID.
+ * @param {string} url - The Steam OpenID login URL to the third party website.
+ * @param {function} callback
+ */
+SteamCommunity.prototype.openidLogin = function(url, callback) {
+	var self = this;
+	self.httpRequestGet({
+		"uri": url,
+		"followAllRedirects": true
+	}, function(err, response, body) {
+		if(err) {
+			callback(err);
+			return;
+		}
+
```

Comment:
It would probably be good to verify that we actually got redirected to steamcommunity.com/openid.